src/js/Lists/ListItem.js

```diff
@@ -460,77 +522,117 @@ export default class ListItem extends PureComponent {
 
     let nestedList;
     if (nestedItems) {
-      nestedList = <Collapse collapsed={!isOpen} animate={animateNestedItems}><List>{nestedItems}</List></Collapse>;
-
-      if (!rightIcon || !rightAvatar) {
-        rightNode = (
-          <TileAddon
-            key="expander-addon"
-            icon={(
-              <Collapser flipped={isOpen} iconClassName={expanderIconClassName}>
-                {expanderIconChildren}
-              </Collapser>
-            )}
-            avatar={null}
-          />
+      if (!cascadingMenu) {
+        nestedList = (
+          <Collapse collapsed={!visible} animate={animateNestedItems}>
+            <List style={nestedListStyle} className={nestedListClassName}>{nestedItems}</List>
+          </Collapse>
         );
       }
+
+      const collapser = (
+        <TileAddon
+          key="expander-addon"
+          icon={
+            <Collapser flipped={visible} iconClassName={expanderIconClassName}>
+              {expanderIconChildren}
+            </Collapser>
+          }
+          avatar={null}
+        />
+      );
+
+      if (expanderLeft) {
+        if (!leftIcon && !leftAvatar) {
+          leftNode = collapser;
+        }
+      } else if (!rightIcon && !rightAvatar) {
+        rightNode = collapser;
+      }
     }
-    const icond = !!leftIcon || !!rightIcon;
+
+    const icond = !!leftIcon || !!rightIcon || !!nestedItems;
     const avatard = !!leftAvatar || !!rightAvatar;
 
-    return (
-      <ItemComponent
-        style={style}
-        className={cn('md-list-item', {
-          'md-list-item--nested-container': nestedItems,
-        }, className)}
-        aria-setsize={ariaSize}
-        aria-posinset={ariaPos}
-        ref={this._setContainer}
+    const tile = (
+      <AccessibleFakeInkedButton
+        {...props}
+        __SUPER_SECRET_REF__={this._setTile}
```

Comment:
Technically don't need this anymore since I figured out how to call refs in cloned elements.. But lazy.